components/forms/Form/Form.tsx

```diff
@@ -45,6 +46,20 @@ const InnerForm = (props: InnerFormProps & FormikProps<FormValues>) => {
     }
   }, [formStatusError, errorList, lastSubmitCount, canFocusOnError]);
 
+  useEffect(() => {
+    // timeout to prevent form from being submitted too quickly
+    const millisPerFormElement = 2000;
+    const formSubmitTimeout = setTimeout(
+      () => {
+        setFormSubmitDelay(true);
+      },
+      children ? React.Children.count(children) * millisPerFormElement : 0
```

Comment:
A possible enhancement would be to filter out rich text components and only use question components to set the timer.

Reply:
Yeah, that's in my comments for this PR. I wasn't sure how accurate the timer needed to be. I also wasn't sure if there were more elements than `richText` elements that should be filtered out.

Reply:
Sorry, I didn't see that question earlier in the 'unresolved questions'.
For now I believe the only elements that aren't interactive are richText and heading.  For this PR it would be nice if we could filter them out as on a lot of the newer forms we're seeing lot's of content in those 2 element types.